ui/App/views/Login.jsx

```diff
@@ -22,7 +22,9 @@ const Login = ({handleLogin}) => {
                 history.push('/');
             }
         } catch (e) {
+            console.log(e);
```

Comment:
Should this be removed?

Reply:
No, i want to keep it in there for future problems.

Reply:
sounds good